account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -0,0 +1,150 @@
+import {
+    custom,
+    keccak256,
+    parseEther,
+    publicActions,
+    toHex,
+    type Address,
+  } from "viem";
+  import { privateKeyToAccount } from "viem/accounts";
+  
+  import { LocalAccountSigner, type SmartAccountSigner } from "@aa-sdk/core";
+  
+  import {
+    createSMAV2SignerAccountClient,
+  } from "./client.js";
+  
+  import { local070InstanceOptSep } from "~test/instances.js";
+  import { setBalance } from "viem/actions";
+  import { accounts } from "~test/constants.js";
+  
+  describe("6900 RI installValidation Tests", async () => {
+    const instance = local070InstanceOptSep;
+    const client = instance.getClient().extend(publicActions);
+
+    const signer: SmartAccountSigner = new LocalAccountSigner(
+      accounts.fundedAccountOwner
+    );
+  
+    it("should send a UO", async () => {
+      // Generate and add the secondary signer
+  
+      const provider = await givenConnectedProvider({ signer })
+  
+      console.log(signer);
+      console.log("provider: ", provider.getAddress());
+
+      await setBalance(instance.getClient(), {
+        address: provider.getAddress(),
+        value: parseEther("2"),
+      });
+    
+      const target = "0x000000000000000000000000000000000000dEaD";
+
+      const startBalance = await client.getBalance({ 
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'startBalance' is assigned a value but never used.
<!-- __reviewdog__:ChA4OWE4Y2Q2YTNlZGQyYmM3EgZyZGpzb24= -->